dataflow/src/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -397,12 +400,14 @@ public void visualizeBlockTransferInput(Block bb, Analysis<A, S, T> analysis) {
 
     @Override
     public void visualizeBlockNode(Node t, /*@Nullable*/ Analysis<A, S, T> analysis) {
-        A value = analysis.getValue(t);
-        String valueInfo = "";
-        if (value != null) {
-            valueInfo = "    > " + prepareString(value.toString());
+        this.sbBlock.append(prepareString(t.toString()) + "   [ " + prepareNodeType(t) + " ]");
+        A value = null;
```

Comment:
If the variable is only used within the if statement, then it should be declared within the if statement, too, in the smallest possible enclosing scope.